.babelrc

```diff
@@ -1,3 +1,6 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react"]
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
```

Comment:
<img width="1139" alt="screen shot 2019-02-17 at 4 44 54 pm" src="https://user-images.githubusercontent.com/43746609/52915371-6aa1fd00-32d3-11e9-975a-742523821919.png">
Do you think adding this plugin to our `webpack config` might help?

Reply:
I don't get you. Help with what?

Reply:
<img width="923" alt="screen shot 2019-02-17 at 4 31 09 pm" src="https://user-images.githubusercontent.com/43746609/52916328-4566bc00-32de-11e9-814b-61d863c10cc7.png">
I initially sent a screenshot of the logs. The app is crashing.

Reply:
I will look into it